doc/classes/InputEventMouseButton.xml

```diff
@@ -66,12 +68,13 @@
 			Mouse button identifier, one of the BUTTON_* or BUTTON_WHEEL_* constants in [@Global Scope].
 		</member>
 		<member name="doubleclick" type="bool" setter="set_doubleclick" getter="is_doubleclick">
-			Whether the event is a double-click.
+			If [code]true[/code] the mouse button's state is a double-click. If [code]false[/code] the mouse button's state is released.
 		</member>
 		<member name="factor" type="float" setter="set_factor" getter="get_factor">
+			TO TALK in PR, reduz said : i think it's used for apple touch but i don't remember what it does
```

Comment:
Hm, this "TO TALK in PR" wasn't noticed, https://github.com/godotengine/godot-docs/issues/1490